dataprofiler/labelers/data_processing.py

```diff
@@ -1328,7 +1332,7 @@ def help(cls):
         print(help_str)
 
     @staticmethod
-    def match_sentence_lengths(data, results, flatten_separator):
+    def match_sentence_lengths(data, results, flatten_separator, inplace=True):
```

Comment:
Need to add the parameter description.

Kinda weird this function occurs twice